src/Abp/Authorization/AuthorizationHelper.cs

```diff
@@ -98,6 +103,21 @@ protected virtual async Task CheckPermissions(MethodInfo methodInfo, Type type)
             await AuthorizeAsync(authorizeAttributes);
         }
 
+        private static bool IsPropertyGetterSetterMethod(MethodInfo method, Type type)
+        {
+            if (!method.IsSpecialName)
+            {
+                return false;
+            }
+
+            if (method.Name.Length < 4)
+            {
+                return false;
+            }
+
+            return type.GetProperty(method.Name.Substring(4), BindingFlags.Instance | BindingFlags.Static | BindingFlags.NonPublic) != null;
```

Comment:
Change 4 to 5. Otherwise, it may throw exception if method.Name.Substring(4) is empty. Didn't try it, but would be safer :)